test/jobs/template_render_job_test.rb

```diff
@@ -21,11 +21,11 @@ def render_job(composer_params)
     it 'render report and delivers it to mail' do
       mailer = mock('mailer')
       mailer.expects('deliver_now')
+      composer_params = { 'foo' => 'bar', 'send_mail' => true, 'mail_to' => 'this@email.cz', 'gzip' => true }
```

Comment:
Hopefully the `this@email.cz` guy does not get a ton of emails from our Jenkins :-)

This is weird, my knowledge of RSpec is low but isn't `expects` supposed to fail the test when not called?

Reply:
It's a mocha spec and indeed it should fail if not called. But if the method is called, it does not invoke the real method. So no e-mails are really sent :)

Reply:
Yeah I understand the concept of mocking, but expects should fail. Do you have any idea why it does not?

Reply:
now because we call the method. Before it failed locally. Why it did not fail on Jenkins is very unclear to me, I'd guess we are not running these tests on Jenkins for some reason.